### PR TITLE
[Merged by Bors] - feat(AlgebraicGeometry): spreading out morphisms on stalks

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/AlgebraicGeometry/SpreadingOut.lean [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+/--
+Let `x : X` and `f g : X ⟶ Y` be two morphisms such that `f x = g x`.
+If `f` and `g` agree on the stalk of `x`, then they agree on an open neighborhood of `x`,
+provided `X` is "germ-injective" (e.g. when it's integral or locally noetherian).
+
+TODO: The condition on `X` is unnecessary when `Y` is locally of finite type.
+-/
+@[stacks 0BX6]
+lemma spread_out_unique_of_germInjective [X.GermInjective] {x : X}
```
Comment (joelriou) [inline Mathlib/AlgebraicGeometry/SpreadingOut.lean:264]: It seems this is a more basic lemma, and that it uses only `X.IsGermInjectiveAt x`, which may also be the take of of the other lemmas above.

Comment (erdOne) [reply Mathlib/AlgebraicGeometry/SpreadingOut.lean:264]: Apart from the `X.IsGermInjectiveAt x` part of the comment which I agree is a great suggestion, I'm not sure what you meant by "a more basic lemma". Are you suggesting that there is a more basic generalization or that it should be moved forwards in the file or something else?

Comment (joelriou) [reply Mathlib/AlgebraicGeometry/SpreadingOut.lean:264]: Yes, the lemma could be moved upwards as it seems more basic.

**Comment by joelriou** [comment]
Thanks!

bors merge

**Pull request closed without merging**